gateway/gateway.go

```diff
@@ -694,7 +701,10 @@ func (gateway *HandleT) getJobDataFromRequest(req *webRequestT) (jobData *jobFro
 	return
 }
 
-func isNonIdentifiable(anonIDFromReq, userIDFromReq string) bool {
+func isNonIdentifiable(anonIDFromReq, userIDFromReq, eventType string) bool {
+	if misc.Contains(EventTypesSkipUserCheck, eventType) {
```

Comment:
There is no need to use `misc.Contains` since `EventTypesSkipUserCheck` is a slice with one element only, right? We can always have a slice later once we have more than one event type that needs skipping.
```suggestion
	if eventType == extractEvent {
```

Reply:
Addressed!